### DISCUSSION TITLE
Mathematical priorities

### DISCUSSION THREAD

**Discussion by alichz2001** [alt-romes/programmer-calculator#51] [Q&A] [answered]
Hello and thanks for this amazing calculator.
this is very helpful but i don't understand why mathematical priorities not considered. it's a bug or feature. please explain me :pray:

**Comment by alt-romes** [comment, marked as answer]
Hello!

1) If you stay tuned it'll soon have mathematical priorities, I'm working on it right now!

2) It isn't either a bug or a feature - it was a decision at the time.
I wanted to get the calculator done as soon as possible, and doing mathematical priorities would be more time consuming and delay the release of a small tool I thought only I would be using. Since this wasn't the case, I'm now writing mathematical priorities into the program.

Thank you for reaching out
~romes

**Reply by alichz2001** [reply]
Thanks for response. Good luck and دمت گرم(Persian). :grin:

**Comment by alt-romes** [comment]
@alichoopani

I've released a new version that implements mathematical priorities and parenthesis for grouping!

I believe you might like it.

Let us know if anything isn't working :)

Thank you,
~romes
